README.md

```diff
@@ -89,14 +91,15 @@ environment variable. This is likely to already be the case by default on a Unix
 Windows it will install to `C:\Program Files (x86)\Corrosion` by default, which will not be in your
 `PATH` or `CMAKE_PREFIX_PATH` by default.
 
-Once Corrosion is installed and you've ensured the package is avilable in your `PATH`, you
+Once Corrosion is installed and you've ensured the package is available in your `PATH`, you
 can use it from your own project like any other package from your CMakeLists.txt:
 ```cmake
 find_package(Corrosion REQUIRED)
 ```
 
 ### FetchContent
-If installation is difficult or not feasible in your environment, you can use the
+If you are using a recent CMake version (>= 3.19) or installation is difficult or not feasible in
```

Comment:
```suggestion
If you are using a CMake >= 3.19 or installation is difficult or not feasible in
```

"recent" is fairly relative, just say what version.

But i'm not sure i understand why there is a `or` here.  Should it be a `and`? (why do we recommand this with CMake .19 but not with earlier version.)

Reply:
> "recent" is fairly relative, just say what version.

Done

Regarding the `or`:  For Cmake >= 3.19 we don't need the native tooling, so the configure is equally fast regardless of corrosion being installed or corrosion being used as a subdirectory. For older CMake versions we need the native generator tool, which increases the configure time significantly if corrosion is not installed.